Formula/clang-format.rb

```diff
@@ -4,10 +4,10 @@ class ClangFormat < Formula
   version "2016-12-09"
 
   stable do
-    url "http://llvm.org/svn/llvm-project/llvm/tags/google/stable/2016-12-09/", :using => :svn
+    url "https://llvm.org/svn/llvm-project/llvm/tags/google/stable/2016-12-09/", :using => :svn
```

Comment:
This is interesting. I actually `brew rm svn` on my machine and then did `brew fetch clang-format` to test this, and it worked. Was definitely using the system `svn` at the time 😕